eelbrain/datasets/_alice.py

```diff
@@ -1,40 +1,32 @@
 # Helper to download data for Alice example
-import hashlib
-import shutil
 from pathlib import Path
-from urllib.request import urlretrieve
-import zipfile
 
-from pooch import file_hash
+import pooch
 
 from .._types import PathArg
 
 
 def get_alice_path(
         path: PathArg = Path("~/Data/Alice"),
 ):
-    md5 = hashlib.md5()
-
     path = Path(path).expanduser().resolve()
-    if path.exists():
-        return path
-    path.mkdir(exist_ok=True, parents=True)
-    urls = [
-        ['https://drum.lib.umd.edu/bitstream/handle/1903/27591/stimuli.zip', '4336a47bef7d3e63239c40c0623dc186'],
-        # ['https://drum.lib.umd.edu/bitstream/handle/1903/27591/eeg.0.zip', 'd63d96a6e5080578dbf71320ddbec0a0'],
-        ['https://drum.lib.umd.edu/bitstream/handle/1903/27591/eeg.1.zip', 'bdc65f168db4c0f19bb0fed20eae129b'],  # S15-S34
-        # ['https://drum.lib.umd.edu/bitstream/handle/1903/27591/eeg.2.zip', '3fb33ca1c4640c863a71bddd45006815'],
-    ]
+    if not path.exists():
+        path.mkdir(exist_ok=True, parents=True)
```

Comment:
```suggestion
    path.mkdir(exist_ok=True, parents=True)
```
`exist_ok=True` takes care of the conditional